templates/app-scaffold/ts/package.json

```diff
@@ -4,7 +4,10 @@
   "description": "",
   "author": "",
   "scripts": {
-    "dev": "nodemon --ext 'js,json,ts' --inspect=0.0.0.0:9229 --exec node -r ts-node/register app.ts",
+    "npm:docker": "docker-compose run kuzzle npm",
```

Comment:
Just `npm`?

Reply:
ah, I guess the purpose of this is to be used by other scripts...

Reply:
Command suffixed with `:docker` are ran inside the container. For example you may want to bump the version, update dependencies, etc